src/containers/Toolbar/index.tsx

```diff
@@ -10,19 +11,32 @@ interface IToolbarProps {
   onFilterValueChange: (filterValue: string) => void;
   preserveLogs: boolean;
   onPreserveLogsChange: (preserveLogs: boolean) => void;
+  onClear: () => void;
 }
 
+const ClearButton = ({ onClick }: { onClick: () => void }) => (
+  <div className="border-0 rounded-md p-2 dark:bg-gray-700 bg-gray-200 opacity-50 outline-none hover:opacity-100">
```

Comment:
The new position looks fine, but could we maintain the same style as other buttons in the toolbar to keep the UI consistent.

The IconButton component already has a hover effect
<img width="896" alt="Screenshot 2021-09-13 at 14 06 07" src="https://user-images.githubusercontent.com/2871038/133088920-43583025-ab5c-4fec-ab94-b2372f85861d.png">